doc/man3/SSL_CTX_set_record_padding_callback.pod

```diff
@@ -32,6 +32,9 @@ SSL_set_block_padding - install callback to specify TLS 1.3 record padding
 SSL_CTX_set_record_padding_callback() or SSL_set_record_padding_callback()
 can be used to assign a callback function I<cb> to specify the padding
 for TLS 1.3 records. The value set in B<ctx> is copied to a new SSL by SSL_new().
+Kernel TLS is not possible if the record padding callback is set. And also the
```

Comment:
Maybe adjust: s/set. And also the/set, and the/.

Reply:
Applied, thanks